### PR TITLE
[stable/grafana] Removed extra schedulerName field

### PR DESCRIPTION
**Pull request by kalpeshdc**
#### What this PR does / why we need it:

There is a repeated `schedulerName` field in the `_pod.tpl` file on lines 3 and 7. If the `Values.schedulerName` is set in the values file then this will create a duplicate entry of the same key i.e. `schedulerName`. I have removed the duplicated `schedulerName` field.

#### Checklist

- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Title of the PR starts with chart name (e.g. `[stable/mychartname]`)

### CODE REVIEW

**Comment by zanhsieh** [comment]
@thekalpesh7 
Could you bump the chart version to latest please? Thank you.

/assign

**Comment by thekalpesh7** [comment]
@zanhsieh I have updated the Chart version to 5.0.14. Let me know If that's okay.

File: stable/grafana/Chart.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: grafana
-version: 5.0.13
+version: 5.0.14
```
Comment (zanhsieh) [inline stable/grafana/Chart.yaml]: Should be 5.0.17 now.

**Review by zanhsieh — CHANGES_REQUESTED** [review]

**Comment by zanhsieh** [comment]
@thekalpesh7 
Could you bump the chart to latest version again?

**Comment by zanhsieh** [comment]
@thekalpesh7 
Could you bump the chart again to latest version please? Thank you.

**Review by zanhsieh — APPROVED** [review]

**Pull request merged**